### PR TITLE
Refactor/changestateobject

### PR DESCRIPTION
**Pull request by dbennin125**
For management's consideration.

### CODE REVIEW

File: src/selectors/markdownSelectors.js [hunk 1]
```diff
@@ -1,5 +1,20 @@
 import marked from 'marked';
 
-export const getMarkdown = state => state.markdown;
+export const getMarkdownID = state => state.currentMarkdownID;
+
+export const getMarkdown = state => {
+  
+  let markdownBody;
+
+  state.markdownList.filter(markdown => {
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.js:9]: use find here instead of filter

File: src/selectors/markdownSelectors.js [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+    if(markdown.id === getMarkdownID(state)) {
+      return markdownBody = markdown.body;
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.js:11]: this is how you would do it if you were using `forEach`.

instead:

```
const markdown = state.markdownList.find(.....);
return markdown?.body;
```

Comment (dbennin125) [reply src/selectors/markdownSelectors.js]: Done.

File: src/selectors/markdownSelectors.js (hunk 1 above)
Comment (dbennin125) [reply src/selectors/markdownSelectors.js]: Done.

**Pull request merged**